aea/cli/registry/fetch.py

```diff
@@ -55,16 +56,19 @@ def fetch_agent(
     :return: None
     """
     author, name, version = public_id.author, public_id.name, public_id.version
-    api_path = f"/agents/{author}/{name}/{version}"
-    resp = request_api("GET", api_path)
-    file_url = resp["file"]
-
-    filepath = download_file(file_url, ctx.cwd)
 
     folder_name = target_dir or (name if alias is None else alias)
     aea_folder = os.path.join(ctx.cwd, folder_name)
+    if os.path.exists(aea_folder):
+        raise ClickException(f'Item "{folder_name}" already exists in target folder.')
```

Comment:
What will be CLI output here? I.e. a full stack trace? We don't want a stack trace, just a single line echo error message.

Reply:
no, ClickException is not printed as a traceback. printed as a message to stderr and exit